### PR TITLE
Make helpwin configurable

### LINKED ISSUE #168: Make help window content configurable
The content of the `HelpWin` can currently only be set / changed by changing the source code / overwriting the component.

It would be very useful to have a configurable way of setting the content (preferably HTML) like we can do for other texts like in the footer.

### PR DESCRIPTION
**Pull request by JakobMiksch**
Makes it possible to control the content of the help window using configuration.

fixes #168

### CODE REVIEW

File: src/components/helpwin/HelpWin.vue [hunk 1]
```diff
@@ -29,18 +32,26 @@
...
-      content: {type: String, required: false, default: '<h3>WebGIS with OpenLayers and Vue.js</h3> Template and re-usable components for webmapping applications with OpenLayers and Vue.js'},
-      infoLink: {type: String, required: false, default: 'https://github.com/meggsimum/wegue'},
-      infoLinkText: {type: String, required: false, default: 'More info'}
+      icon: {type: String, required: false, default: 'help'}
     },
     data () {
       return {
-        show: false
+        show: false,
+        title: '',
```
Comment (chrismayer) [inline src/components/helpwin/HelpWin.vue:40]: I think the apply of the config data can be done directly `title: this.$appConfig.modules['wgu-helpwin'].windowTitle`, so implementing the `mounted` function could be obsolete.

Comment (JakobMiksch) [reply src/components/helpwin/HelpWin.vue]: that's right. 
I added directly assigned the configs in the  `data` property

**Comment by chrismayer** [comment]
`app-starter/app-conf-projected.json` and `app-starter/app-conf-minimal.json` should also get the settings for the help window.

**Comment by JakobMiksch** [comment]
@chrismayer I added the helpwin to `minimal` and `projected` as well.

**Review by chrismayer — APPROVED** [review]
LGTM, thanks!

**Pull request merged**